reference_data/management/commands/update_gene_shet.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from reference_data.management.commands.utils.update_utils import GeneCommand, ReferenceDataHandler
+from reference_data.models import GeneShet
+
+logger = logging.getLogger(__name__)
+
+
+class ShetReferenceDataHandler(ReferenceDataHandler):
+
+    model_cls = GeneShet
+    url = 'https://storage.googleapis.com/seqr-reference-data/Shet/Shet_Zeng_2023.tsv'
```

Comment:
add a comment in this file explaining how that file was generated/ where it came from